docs/checks.md

```diff
@@ -566,9 +566,22 @@ and the required write permissions are declared at the
 One point is reduced from the score if all jobs have their permissions defined but the top level permissions are not defined. 
 This configuration is secure, but there is a chance that when a new job is added to the workflow, its job permissions could be 
 left undefined because of human error.
-        
+
 The check cannot detect if the "read-only" GitHub permission setting is
-enabled, as there is no API available.   
+enabled, as there is no API available.
+
+Additionally, points are reduced if certain write permissions are defined for a job. 
+
+### Write permissions causing a small reduction
+* `statuses` - May allow an attacker to change the result of pre-submit checks and get a PR merged.
+* `checks` - May allow an attacker to remove pre-submit checks and introduce a bug.
+* `security-events` - May allow an attacker to read vulnerability reports before a patch is available. However, points are not reduced if the job utilizes a recognized action for uploading SARIF results.
+* `deployments` - May allow an attacker to charge repo owner by triggering VM runs, and tiny chance an attacker can trigger a remote service with code they own if server accepts code/location variables unsanitized.
+
+### Write permissions causing a large reduction
+* `contents` - Allows an attacker to commit unreviewed code. However, points are not reduced if the job utilizes a recognized packaging action or command.
+* `packages` - Allows an attacker to publish packages. However, points are not reduced if the job utilizes a recognized packaging action or command.
+* `actions` - May allow an attacker to steal GitHub secrets by adding a malicious workflow or action.
```

Comment:
do you mind double checking the `actions` permissions (I ought to take a closer look but have not fond the time to do so)? I don't think it allows adding a workflow, for example. I think it allows running an existing workflow, which could by pass the LGTM that is used before running workflows on pull request.

Reply:
@chrismcgehee -- Ping on Laurent's q above

Reply:
If https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#permission-on-actions exhaustively lists what can be done with the `actions:write` permission, then it allows:
* Deleting artifacts from a previous action run
* Approving to run an action that needs approval
* Canceling an action that's currently running
* Deleting all logs from a workflow run
* Re-run a workflow that was previously run

I think approving an action may allow an attacker to steal GitHub secrets. Does that seem like enough to keep it as a high risk permission?

Reply:
I'd say we can keep it as high for now, since secrets typically give access to resources like ppg keys, cloud services, etc. If we get complaints of false positives, we could double-check that there is at least one workflow that uses a secret in the repo.
Wdut?

Reply:
That sounds good. I left the scoring the same but updating the verbiage for the documentation.